app/views/pages/sponsors.volt

```diff
@@ -24,15 +24,31 @@
                 <h3>{{ locale.translate('sponsors_and_supporters_upper') }}</h3>
                 <a href="https://www.pdffiller.com/" target="_blank">
                     <img width="240px"
-                         src="https://raw.githubusercontent.com/phalcon/cphalcon/master/backers/pdffiller-240x60.png">
+                         src="https://raw.githubusercontent.com/phalcon/cphalcon/master/backers/pdffiller-240x60.png" alt="Pdffiller">
                 </a>
                 <a href="https://mctekk.com/phalcon" target="_blank">
                     <img width="240px"
-                         src="https://raw.githubusercontent.com/phalcon/cphalcon/master/backers/mctekk-240x60.png">
+                         src="https://raw.githubusercontent.com/phalcon/cphalcon/master/backers/mctekk-240x60.png"  alt="Mctekk">
                 </a>
                 <a href="https://abits.com/" target="_blank">
                     <img width="240px"
-                         src="https://raw.githubusercontent.com/phalcon/cphalcon/master/backers/abits-240x60.png">
+                         src="https://raw.githubusercontent.com/phalcon/cphalcon/master/backers/abits-240x60.png" alt="Abits">
```

Comment:
Can you see if we can change the URL to the assets domain?

Reply:
@niden done